File: ios/Shared/Const.swift
import Foundation

enum Const {
    static let widgetKind = "com.mizuki.Ohashi.Pilll.widget"

    static let userIsPremiumOrTrial = "userIsPremiumOrTrial"

    static let pillSheetTodayPillNumber = "pillSheetTodayPillNumber"
    static let pillSheetEndDisplayPillNumber = "pillSheetEndDisplayPillNumber"
    // Epoch milli second
    static let pillSheetLastTakenDate = "pillSheetLastTakenDate"
    // Epoch milli second
    static let lastRecordDate = "lastRecordDate"
}

enum Plist {
    static var appGroupKey: String {
        Bundle.main.infoDictionary!["AppGroupKey"] as! String
    }
}

File: android/app/src/main/kotlin/com/example/Pilll/Const.kt
package com.mizuki.Ohashi.Pilll

abstract class Const {
    companion object {
        const val userIsPremiumOrTrial = "userIsPremiumOrTrial"

        const val pillSheetGroupTodayPillNumber = "pillSheetGroupTodayPillNumber"
        const val pillSheetTodayPillNumber = "pillSheetTodayPillNumber"
        const val pillSheetEndDisplayPillNumber = "pillSheetEndDisplayPillNumber"

        // Epoch milli second
        const val pillSheetLastTakenDate = "pillSheetLastTakenDate"

        // Epoch milli second
        const val pillSheetValueLastUpdateDateTime = "pillSheetValueLastUpdateDateTime"

        const val settingPillSheetAppearanceMode =
            "settingPillSheetAppearanceMode" // number or date or sequential
    }
}